skbio/sequence/_grammared_sequence.py

```diff
@@ -210,10 +312,11 @@ def _validate(self):
                 invalid_characters > 0)[0].astype(np.uint8).view('|S1'))
             raise ValueError(
                 "Invalid character%s in sequence: %r. \n"
-                "Lowercase letters are not used in IUPAC notation. You can "
+                "Note that lowercase letters are not valid by default. "
```

Comment:
There isn't really a default, how about the following?

*Use `lowercase` if your sequence contains lowercase characters not in the sequence's alphabet.*